.github/workflows/performance_tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Performance Tests
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
As a future extension of this project, what do you think about adding a comment to the PR with a markdown table of the perf results? This will last whereas the artifact will go away after 90 days.

Reply:
Yeah, this is a cool idea. The test script can also be given a flag to specify markdown output to make it easier.

Reply:
I love this! I already have [a ticket](https://github.com/newrelic/newrelic-ruby-agent/issues/1395) out to enhance notifications and will add this note to it :)